pkg/crc/api/helpers.go

```diff
@@ -45,53 +66,81 @@ func (c *context) Code(code int) error {
 }
 
 type server struct {
-	routes     map[string]map[string]func(*context) error
+	routes     map[string]map[string]map[apiVersion]func(*context) error
 	routesLock sync.RWMutex
 }
 
 func newServer() *server {
 	return &server{
-		routes: make(map[string]map[string]func(*context) error),
+		routes: make(map[string]map[string]map[apiVersion]func(*context) error),
 	}
 }
 
-func (s *server) GET(pattern string, handler func(c *context) error) {
+func (s *server) GET(pattern string, handler func(c *context) error, version apiVersion) {
+	if !version.isValid() {
+		panic("Invalid version used to register handler")
```

Comment:
have to remove these panics

Reply:
panics are bad. throw errors, log, but panic is too strong

Reply:
to give more context: the reason i went with panic is because this catches it while running the unit tests if a wrong version is used to register the handler